### ISSUE TITLE
Virus detection by VirusTotal (Trojan)

### ISSUE THREAD

**Issue by eXenz** [Ryochan7/DS4Windows#2477]
When analyzing the program files with virustotal the archives get flagged as trojans called "Trojan.Malware.300983.susgen" and "Trojan.Bladabindi.Win32.139744"
Are those false positives or should I be worried about it? This was detected by MaxSecure and Zillya respectively. Windows Defender didn´t detect anything.
Is the program safe to download?

**Comment by Masamune3210** [comment]
Yes, there is no malware of any kind in this project

**Comment by Ryochan7** [comment]
TLDR: Need a minimum $80 USD / year to avoid this problem

At least "Trojan.Malware.300983.susgen" has been a known false positive for years based on results from Google. The other result only shows up in one archive of the bunch and I doubt it is real either. VirusTotal is completely untrustworthy and is pretty much an extortion racket. If I could still digitally sign the binaries I am sure ViruslTotal would not complain at all; that was the case with later 3.0.x releases when I was able to purchase a 1 year OV cert and properly sign DS4Windows binaries with it. Seen reports from other open source projects that have documented similar issues. In the current year, even developing end user desktop software for Windows is a pay to play endeavor. 

Definitely not open source friendly if you are not beholden to a corporate entity

**Comment by Masamune3210** [comment]
Its unfortunately not any better on the Apple side of the fence either, the only truly free way to make programs anymore is to make them for a Unix-like, but that severely limits your user base

**Comment by ShaunTheQuietGamer** [comment]
> VirusTotal is completely untrustworthy and is pretty much an extortion racket. If I could still digitally sign the binaries I am sure ViruslTotal would not complain at all

I would disagree. VirusTotal is a great utility if you use it right. [This](https://www.virustotal.com/gui/file/632e9208865b2d5ddeb2ae6adb685a69101aed5fd0e65a1a220ff580e7e4c224?nocache=1) is a scan for the most recent version of DS4Windows. There is _one_ detection, that's it. Its pretty safe to assume that file is virus-free. Just because one antivirus says its a virus doesn't instantly mean it is; its not a test that a file has to get 0/61 on to be considered virus-free. The idea is to get a bigger picture by running a file against a bunch of AVs, and using that information to decide if a file is like a virus or not. If a file gets 50 positives for a certain virus, its almost guaranteed to contain that virus. The closer you get to 0 positives, the more likely the file is safe.
